Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -32,6 +32,20 @@ open class MessageComposerInputAccessoryViewController<ExtraData: ExtraDataTypes
         }
     }
     
+    public var threadParentMessage: _ChatMessage<ExtraData>? {
+        didSet {
+            guard let memberCount = controller?.channel?.memberCount else { return }
+            
+            composerView.setCheckmarkView(hidden: false)
+            
+            if memberCount > 2 {
+                composerView.checkmarkView.label.text = L10n.Composer.Checkmark.channelReply
+            } else {
+                composerView.checkmarkView.label.text = L10n.Composer.Checkmark.directMessageReply
+            }
```

Comment:
Shouldn't this happen in `updateContent` and we should rather call `updateContentIfNeeded` here?